src/propsheet/fontdlg.cpp

```diff
@@ -685,7 +686,10 @@ FontListCreate(
     /*
      * This only enumerates face names and font sizes if necessary.
      */
-    EnumerateFonts(bLB ? EF_OEMFONT : EF_TTFONT);

```

Comment:
Uncertain about this.

Reply:
This seems right - looks like EnumerateFonts can only fail if it fails to HeapAlloc some memory, so we can probably just bail here when that's the case.